docs/install/uninstallEverest.md

```diff
@@ -1,53 +1,59 @@
 # Uninstall Everest
 
-No matter why you need to uninstall Everest, you can run the commands below to remove all the Everest resources including: 
+No matter why you need to uninstall Everest, you can run the commands below to remove all the Everest resources including:
 
 - All Kubernetes objects created by Everest
 - All docker containers created by Everest
 - All downloaded binaries and files like **everestctl**, **quickstart.yml**.
 
-
+!!! note alert alert-primary "Warning"
+  Performing these actions will remove all database clusters and data from the Kubernetes cluster
```

Comment:
Can also mention, Please validate before deleting any CRD as deleting the CRD  will break the dependent custom resources (if any) present on the cluster